specification/maps/data-plane/Search/preview/1.0/search.json

```diff
@@ -768,7 +768,7 @@
     "/search/poi/category/tree/{format}": {
       "get": {
         "description": "**Get POI Category Tree**\n\n**Applies to**: S0 and S1 pricing tiers.\n\n\nPOI Category API provides a full list of supported Points of Interest (POI) categories and subcategories together with their translations and synonyms. The returned content can be used to provide more meaningful results through other Search Service APIs, like [Get Search POI](https://docs.microsoft.com/rest/api/maps/search/getsearchpoi).",
-        "operationId": "Search_GetSearchPOICategoryTreePreview",
+        "operationId": "GetSearchPOICategoryTreePreview",
```

Comment:
Is this still a Preview? I don't think so...

Reply:
Removed "Preview"